hdr_schemata/models/SchemaOrg/GoogleRecommended/__init__.py

```diff
@@ -0,0 +1,215 @@
+        example= [
+            {
+                "@type": "Person",
+                "sameAs": "https://orcid.org/0000-0000-0000-0000",
+                "givenName": "Jane",
+                "familyName": "Foo",
+                "name": "Jane Foo"
+            },
+            {
+                "@type": "Person",
+                "sameAs": "https://orcid.org/0000-0000-0000-0001",
+                "givenName": "Jo",
+                "familyName": "Bar",
+                "name": "Jo Bar"
+            },
+            {
+                "@type": "Organization",
+                "sameAs": "https://ror.org/xxxxxxxxx",
+                "name": "Fictitious Research Consortium"
+            }
+        ]
+    )
+
+    #overload with google examples and description
+    citation: Optional[Union[Text,CreativeWork]] = Field(
+        None,
+        title='citation',
+        description="Identifies academic articles that are recommended by the data provider be cited in addition to the dataset itself. Provide the citation for the dataset itself with other properties, such as name, identifier, creator, and publisher properties. For example, this property can uniquely identify a related academic publication such as a data descriptor, data paper, or an article for which this dataset is supplementary material for. Don't use this property to provide citation information for the dataset itself. It is intended to identify related academic articles, not the dataset itself. To provide information necessary to cite the dataset itself use name, identifier, creator, and publisher properties instead.",
+        examples=[
+            "https://doi.org/10.1111/111",
+            "https://identifiers.org/pubmed:11111111",
+            "https://identifiers.org/arxiv:0111.1111v1",
+            "Doe J (2014) Influence of X ... https://doi.org/10.1111/111"
+        ]
+    )
+
+    #overload with google examples
+    #funder: Optional[Union[Person,Organization]] = Field(
+    funder: Optional[Organization] = Field(
+        None,
+        title='Funder',
+        description='A person or organization that provides financial support for this dataset. To uniquely identify individuals, use ORCID ID as the value of the sameAs property of the Person type. To uniquely identify institutions and organizations, use ROR ID.',
+        examples=[
+            {
+                "@type": "Person",
+                "sameAs": "https://orcid.org/0000-0000-0000-0002",
+                "givenName": "Jane",
+                "familyName": "Funder",
+                "name": "Jane Funder"
+            },
+            {
+                "@type": "Organization",
+                "sameAs": "https://ror.org/yyyyyyyyy",
+                "name": "Fictitious Funding Organization"
+            }
+        ]
+    )
+
+    #not implemented in the "base" yet...
```

Comment:
Just for clarity on these fields that aren't implemented in "base" yet, are these instances where the "google recommended" schema doesn't match with schema.org?

Reply:
actually, i'm just not 100% sure yet. So this was a note to self .. when i move onto a future ticket of implementing the full schema.org in pydantic, i should to move these to the 'base' if they're no different